src/test/java/plugins/GitPluginTest.java

```diff
@@ -209,6 +210,142 @@ public void update_submodules_recursively() throws IOException, InterruptedExcep
         job.startBuild().shouldSucceed();
     }
 
+    @Test
+    public void calculate_changelog() throws IOException, InterruptedException, SftpException, JSchException {
```

Comment:
throws can be removed now (since https://github.com/jenkinsci/acceptance-test-harness/commit/220c16baf0b4320eff0b59471d7d0bfe929f13b7)